pillar/journal-public.sls

```diff
@@ -20,6 +20,8 @@ journal:
     # - https://developers.google.com/search/reference/robots_txt#url-matching-based-on-path-values
     # 'crawl-delay' directive:
     # - https://en.wikipedia.org/wiki/Robots_exclusion_standard#Crawl-delay_directive
+    # lsh@2021-01-04: bingbot ignores rules that apply to all bots (*) if it has a specific entry.
+    # lsh@2021-02-09: robots.txt has no proper specification! https://github.com/elifesciences/issues/issues/6451
     robots:
         - |
             User-agent: *
```

Comment:
Couple of considerations...
1. Looking at https://elifesciences.org/robots.txt, seems the `$robots_disallow` isn't expanding to anything. It's not from the spec an issue are far as I can tell, as crawlers should just ignore empty `Disallow` fields. But maybe if it isn't used we should remove it and just stick to defining everything statically here?
2. Since the behaviour isn't additive, should we also replicate the `Crawl-delay` field under each of the bot entries? Well all except `SemrushBot` which we block entirely?
3. Reading through the proposed spec by Google, seems this won't cover any subdomains. Is that something that we need to consider following up on?

Reply:
1. not sure what is going on with that, it's some nginx wrangling that Chris did. It seems to have a value in continuumtest: https://continuumtest--journal.elifesciences.org/robots.txt
2. `Crawl-delay` is non-standard, whatever that means in the case of robots.txt, and only Bingbot obeys it. Google have a webmastertools widget you can slide around to alter how often they crawl you but they recommend you leave it to 'auto'.
3. subdomains in elife's case are individual apps/services. They shouldn't come under the journal's robots.txt purview

Reply:
Regarding point 1, it should be investigated at some point, I wouldn't even mind a simple plain text robots.txt file, but I think this can be merged without it for now.